templates/create-branches.sh

```diff
@@ -0,0 +1,15 @@
+
+if [ $1 ]; then 
+  for element in "${templates[@]}"
+  do
+      git clone "$github/$element"
+      cd $element
+      
+      git checkout -b "$1"
+      git push --set-upstream origin "$1"
+      cd ../
+      rm -rf $element
+  done
+else 
+  echo "You must pass the branch name you want to create"
+fi
```

Comment:
I find this unexpected.

I would prefer `create-branches.sh` to declare a function I can later call. If that were the case, `create-branches-play.sh` above would become:

```bash
#!/bin/bash
. templates-list-play.sh
. lib-create-branches.sh
createBranches(playTemplatesList)
```

This way, the script creating the list is decoupled a bit from the script using the list (current implementation enforces the list name).